config/overlays/mutation_webhook/webhook.yaml

```diff
@@ -0,0 +1,27 @@
+# TODO: this is a temporary placement for the mutation webhook
+# It is kept separate until the mutation feature is stable enough.
+# Then the mutation webhook should be moved to config/webhooks/manifests.yaml  
+
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: gatekeeper-mutating-webhook-configuration
+webhooks:
+- clientConfig:
+    caBundle: Cg==
+    service:
+      name: webhook-service
+      namespace: system
+      path: /v1/mutate
+  failurePolicy: Ignore
+  name: mutation.gatekeeper.sh
+  rules:
+  - apiGroups:
+    - '*'
+    apiVersions:
+    - '*'
+    operations:
+    - CREATE
```

Comment:
no update?

Reply:
Good catch, we should call this webhook on update as well

Reply:
Done